common/build.gradle

```diff
@@ -21,8 +21,8 @@ dependencies {
 
 jacocoTestReport {
     reports {
-        xml.enabled false
-        csv.enabled false
+        //xml.enabled false
+        //csv.enabled false
```

Comment:
You might need to add a `html.required = true` in reports part.

Reply:
will add, thanks